core/chains/evm/txmgr/helpers_test.go

```diff
@@ -9,3 +9,7 @@ func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EthCon
 func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EthBroadcaster) {
 	ethBroadcaster.resumeCallback = resumeCallback
 }
+
+func ResendUnconfirmed(er *EthResender) error {
```

Comment:
I think you are allowed to define this as method but nbd obviously :shrug: 
```suggestion
func (er *EthResender) ResendUnconfirmed() error {
```